src/views/monitoring/alert-manager/lib/helper.ts

```diff
@@ -1,9 +1,24 @@
-import { ALERT_STATE_FILTER, SCOPE } from '@/views/monitoring/alert-manager/lib/config';
+import { ALERT_STATE, SCOPE } from '@/views/monitoring/alert-manager/lib/config';
 
 export const alertStateBadgeStyleTypeFormatter = (alertState) => {
-    if (alertState === ALERT_STATE_FILTER.TRIGGERED) return 'red100';
-    if (alertState === ALERT_STATE_FILTER.ACKNOWLEDGED) return 'blue200';
-    return 'gray200';
+    let style = '';
+    switch (alertState) {
+    case ALERT_STATE.TRIGGERED:
+        style = 'red100';
+        break;
```

Comment:
```suggestion
    case ALERT_STATE.TRIGGERED: style = 'red100'; break;
```
이렇게 한번 제안드려봅니다.ㅎㅎ

switch 문 쓸 때 짧으면 걍 한줄로 하는게 가독성이 저는 더 좋더라구요!
근데 사람마다 다 다를 것 같아요. 
switch 문 사용에 대해서도 호불호 갈리긴 하는데 논의해보면 좋을 것 같아요!
1. 간단한 매칭 조건문인 경우, 3개 이상 되면 if else 문 말고 switch 문 쓴다. (찬/반)
2. switch 문에서 각 case 문은 처리할 내용이 1줄 정도의 짧은 내용이면 제안한 내용처럼 case, 처리할 내용, break 를 모두 한 줄로 작성한다. (찬/반)

다른 분들은 각각 어떤거 선호하세요??ㅎ

Reply:
javascript 에서 if/else vs switch 어떻게 처리하는지 좀 더 알아보고 의견 나눠도 좋겠네요! 이럴 때 공부하는거죠 ㅎ

Reply:
1은 찬성이고, 2는 좀 애매해요 ㅠㅠ 오히려 좀 길어도 위에꺼가 가독성이 좋게 느껴져서요..!

Reply:
조곰만 시간을 주시면..!! 리서치해보겠습니다 ㅎㅎ

Reply:
전 if/else 선호하는 편인데 사실 3개 이상 넘어가는 경우가 잘 없어서, 그때 switch 쓰면 좋겠네요! 물론 말씀하신 대로 간단한 매칭 조건문일 때!
그리고 2번은 뭐든 상관은 없는데 case 들이 통일되면 하면 좋겠어요! 다 1줄로 작성하거나 다 정석(?)으로 쓰거나 이렇게요

Reply:
1번에는 찬성입니다~~! https://aahc.tistory.com/6 요거 보고 switch로 바꿨었욤!
2번은 저는 조금 길게 쓰는게 가독성이 좋게 느껴져요~!